docs/content/Getting-Started/Getting-Started-Docker.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Docker
+permalink: /getting-started/docker
+category: Getting Started
+menuOrder: 2
+redirect_from:
+  - /getting-started-docker
+---
+
+This guide will help you get Cube.js running using Docker.
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| Prefer using Docker Compose?
+| [Check out this page instead](/getting-started/docker/compose).
+<!-- prettier-ignore-end -->
+
+> LOOM VIDEO HERE
+
+## 1. Create a new project
+
+In a new folder for your project, run the following command:
+
+```bash
+docker run -p 4000:4000 -v cube:/cube/conf -e CUBEJS_DEV_MODE=true cubejs/cube
+```
+
+## 2. Open Developer Playground
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| This step assumes you can connect to a database instance. If you're unable
+| to connect to a remote instance, please use a Docker image to run one in
+| your local development environment.
+<!-- prettier-ignore-end -->
+
+Head to [http://localhost:4000](http://localhost:4000) to open [Developer
+Playground][ref-devtools-playground].
+
+The [Developer Playground][ref-devtools-playground] has a database connection
+wizard that loads when Cube.js is first started up and no `.env` file is found.
+
+<div
+  style="text-align: center"
+>
+  <img
+  src="https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Getting-Started/connection-wizard-1.png"
+  style="border: none"
+  width="100%"
+  />
+</div>
+
+Click on the type of database to connect to, and you'll be able to enter
+credentials:
+
+<div
+  style="text-align: center"
+>
+  <img
+  src="https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Getting-Started/connection-wizard-2.png"
+  style="border: none"
+  width="100%"
+  />
+</div>
+
+After clicking Apply, you should see tables available to you from the configured
+database. Select one to generate a data schema. Once the schema is generated,
+you can execute queries on the Build tab.
+
+## Next Steps
+
+Generating Data Schema files in Developer Playground is a good first step to
+start modelling your data. You can [learn more about Cube.js Data
+Schema][ref-cubejs-schema] for complex data modelling techniques.
+
+Learn how to [query Cube.js with REST API][ref-rest-api] or [use Javascript
+client library and integrations with frontend
+frameworks][ref-frontend-introduction].
+
+### Configuration with cube.js file
```

Comment:
I believe in the example above we recommend mounting the whole directory (with schema cube.js .env etc). I guess here we can just reinforce it and not advise to mount things separately.